### PR TITLE
Update ps_emailsubscription.tpl

### LINKED ISSUE #29018: blocknewsletter - In FO block is not centered anymore
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

In develop and 8.0.x ❌ 
Screenshot 2022-07-08 at 14 58 08

In 1.7.8.x ✅ 
Screenshot 2022-07-08 at 14 59 12

### Expected behavior

Block newsletter should be centered

### Steps to reproduce

- Go to FO
- scroll down
- see block not centered

### PrestaShop version(s) where the bug happened

8.0.x, develop

### PHP version(s) where the bug happened

7.3

### If your bug is related to a module, specify its name and its version

blocknewsletter v2.0.3

### PR DESCRIPTION
**Pull request by LouisAUTHIE**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Corrects the centering issue on prestashop 8.0.x
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/29018
| How to test?      | Go to front office index page
| Possible impacts? | Nothing

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by Hlavtox — CHANGES_REQUESTED** [review]
Hi @LouisAUTHIE, thank you for your contribution, however, it must be done differently. This code is wrong on multiple levels.

`<div class="block_newsletter container col-lg-8 col-md-12 col-sm-12" id="blockEmailSubscription_{$hookName}">`

- `container `class should be used only for limiting the screen width. 
- `col-lg-8 col-md-12 col-sm-12` are used wrongly and without a `row `class. These classes should be used for responsivity, not limiting the width.

This is a block that can be shown anywhere and it should be styled by CSS for the specific hook.

Ping @NeOMakinG

**Comment by ghost** [comment]
One of the problems is that block social is not displayed because there is no url for the social networks. 

We could add urls to the installation of the social module.

So the https://github.com/PrestaShop/PrestaShop/issues/29018 is not an issue ?

**Comment by LouisAUTHIE** [comment]
I have done a new proposition with a style modification that should do the same but cleaner. Thanks guys for the reviews.

File: _dev/css/components/footer.scss [hunk 1]
```diff
@@ -2,6 +2,11 @@
   padding-top: 2.5rem;
 }
 
+#blockEmailSubscription_displayFooterBefore{
+  margin: auto;
+  float:none;
```
Comment (NeOMakinG) [inline _dev/css/components/footer.scss]:
```suggestion
  float: none;
```

File: _dev/css/components/footer.scss [hunk 2]
```diff
@@ -2,6 +2,11 @@
   padding-top: 2.5rem;
 }
 
+#blockEmailSubscription_displayFooterBefore{
+  margin: auto;
+  float: none;
```
Comment (NeOMakinG) [inline _dev/css/components/footer.scss]:
```suggestion
  float: none;
  margin: auto;
```

**Comment by NeOMakinG** [comment]
Hey @LouisAUTHIE and thanks for the contribution!

@Hlavtox is probably right, this block shouldn't be centered everywhere as it can be used in any hooks!

I guess even the `margin: auto;` isn't a good practice, do we want to center the block inside any hook like in a sidebar/column?

**Comment by matks** [comment]
@LouisAUTHIE Hello, will you continue this topic or shall we take it 😉 ?

**Comment by matks** [comment]
Hi,

Since we had no news from you for more than 30 days, I will follow [our guidelines](https://devdocs.prestashop.com/1.7/project/maintainers-guide/reviewing-pull-requests/#stale-pull-requests) and I'll close this pull request as it seems stale. Feel free to reopen or open another one if you think it's still relevant.

Thanks!

**Pull request closed without merging**